QEfficient/utils/_utils.py

```diff
@@ -67,6 +67,15 @@ def hf_download(
             else:
                 raise e
 
+        except OSError as e:
+            logger.error(f"OSError: {e}")
+            if "Consistency check failed" in str(e):
+                logger.info(
+                    "OSError: Consistency check failed: file should not be incomplete, Resuming the downloading..."
```

Comment:
rephrase this to something like "Consistency check failed during model download. The file appears to be incomplete. Resuming the download..."

Reply:
will do.